README.md

```diff
@@ -18,123 +14,98 @@
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
-  - [Usage](#usage)
   - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
+  - [Deployment](#deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-#📖 [math magician] <a name="about-project"></a>
+## 📖 Overview <a name="about-project"></a>
 
 **[math magician]** is a Single Page Application (SPA) that allows users to make simple calculations, generate and read random math-related quotes and also tweet the quote.
 
-## 🛠 Built With <a name="built-with"></a>
-
-- React 
-- Linters
+### Project Objectives <a name="project-objectives"></a>
 
-### Tech Stack <a name="tech-stack"></a>
+- [x] set up the environment and tools needed to develop a React application
 
-> 
+## 🛠 Built With <a name="built-with"></a>
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
+### Tech Stack <a name="tech-stack"></a>
 
-<!-- Features -->
+- [react](https://react.dev/)
+- [npm](https://www.npmjs.com/)
+- [Git](https://git-scm.com/)
+- [GitHub](https://github.com)
+- [Stylelint](https://stylelint.io/)
+- [ESLint](https://eslint.org/)
+- [sass](https://sass-lang.com/)
+- [VSCode](https://code.visualstudio.com/)
+- [react router](https://reactrouter.com/)
 
 ### Key Features <a name="key-features"></a>
 
->Perform mathmatical calculations
+- [x] Setting up a react app.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] The Live demo link is not required for right now. You should remove it so it looks professional.